pkg/managed/namespace_test.go

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package managed
+
+import (
+  "context"
+  "github.com/stretchr/testify/assert"
+  "github.com/verrazzano/verrazzano-operator/pkg/types"
+  "github.com/verrazzano/verrazzano-operator/pkg/util"
+  v1 "k8s.io/api/core/v1"
+  metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+  "testing"
+)
+
+func TestCreateNamespaces(t *testing.T) {
+  type args struct {
+    mbPair              *types.ModelBindingPair
+    filteredConnections map[string]*util.ManagedClusterConnection
+  }
+  tests := []struct {
```

Comment:
might want to make args struct at the file level since it is duplicated a few times

Reply:
That doesn't work since each uses the args defined directly above it.  Each one has a different set of args.

Reply:
It is a struct so i don't follow why it cannot be reused.  I am not talking about the variable.